_includes/tools/ava/install.md

```diff
@@ -0,0 +1,3 @@
+```sh
```

Comment:
`sh` => `console`

`$` makes it not a shell script. `console` works fine though.